File: fluentcore/src/main/java/com/mcreater/fxfluent/xaml/tags/StaticResourceRedirectContentTag.java
package com.mcreater.fxfluent.xaml.tags;

public class StaticResourceRedirectContentTag extends SimpleContentTag<String> {
    public String toObject() {
        return element().attributeValue("ResourceKey");
    }

    public String toString() {
        return "Redirection {" + super.toString() + "}";
    }
}

File: fluentcore/src/main/java/com/mcreater/fxfluent/xaml/tags/StaticResourceRedirectContentTag.kt
package com.mcreater.fxfluent.xaml.tags


class StaticResourceRedirectContentTag : SimpleContentTag<String>() {
    override fun toObject(): String {
        return element().attributeValue("ResourceKey")
    }

    override fun toString(): String {
        return "Redirection {" + super.toString() + "}"
    }
}

